subsys/bluetooth/mesh/beacon.c

```diff
@@ -334,6 +334,13 @@ static void secure_beacon_recv(struct net_buf_simple *buf)
 
 	cache_add(data, sub);
 
+	kr_change = bt_mesh_kr_update(sub, BT_MESH_KEY_REFRESH(flags), new_key);
+	if (kr_change) {
```

Comment:
I wonder if I've then read the spec wrong (or perhaps the spec was changed?) about reacting to non-primary subnets. It might make sense to include a spec section reference in the commit message, and perhaps also a quote from the text there.

Reply:
I added a quote from the spec in the commit message.